modules/yieldlabBidAdapter.js

```diff
@@ -142,6 +142,24 @@ export const spec = {
           }
         }
 
+        if (isNative(bidRequest, adType)) {
+          const url = `${ENDPOINT}/d/${matchedBid.id}/${bidRequest.params.supplyId}/?ts=${timestamp}${extId}${gdprApplies}${gdprConsent}${pvId}`
+          bidResponse.adUrl = url
+          bidResponse.mediaType = NATIVE
+          const nativeImageAsset = matchedBid.native.assets.find(e => e.id === 2) ? matchedBid.native.assets.find(e => e.id === 2).img : {url: '', w: 0, h: 0};
+          bidResponse.native = {
+            title: matchedBid.native.assets.find(e => e.id === 1) ? matchedBid.native.assets.find(e => e.id === 1).title.text : '',
+            body: matchedBid.native.assets.find(e => e.id === 3) ? matchedBid.native.assets.find(e => e.id === 3).data.value : '',
+            image: {
```

Comment:
Do we support image assets only? What about video assets?

Reply:
Same thought, I did not spot in code where we return video native assets.